vtex-apps/product-price
# feat: added price with tax without unit multiplier

#### What problem is this solving?
I've added a new handle for `sellingPriceWithTaxWithoutUnit` and `listPriceWithTaxWithoutUnit` that allows the display of a product that has a unit multiplier and also taxes. The default `sellingPriceWithTax` calculates the taxes for the entire quantity, not for 1 unit.

#### How to test it?
You can install `vtex.product-price@1.31.1-beta.1` and change the price block configuration on a PDP. The product you are viewing must have a unit multiplier different from 1 and also active taxes.

You can check out this product that has a 5,0 unit multiplier, 100 ron price and a 10% tax.
You can notice the calculation using the default {sellingPriceWithTax} will display the price for 1 unit but the tax for all 5 units
https://vtexromania.myvtex.com/rama-foto-argintata-jardin-d-eden-christofle/p 

<img width="969" alt="Screenshot 2023-06-15 at 13 18 52" src="https://github.com/vtex-apps/product-price/assets/84322200/53e1ba5c-83d8-4ffa-aa70-d2ffbda3ff32">


Checking the new sellingPriceWithTaxWithoutUnit you will see the correct calculation for 1 unit + 1 unit tax
https://mihaitest--vtexromania.myvtex.com/rama-foto-argintata-jardin-d-eden-christofle/p 

<img width="969" alt="Screenshot 2023-06-15 at 13 19 02" src="https://github.com/vtex-apps/product-price/assets/84322200/ea9f2d0b-92f7-4007-b883-e8d180e74133">
